src/styles/_card.scss

```
@@ -0,0 +1,83 @@
+.card {
+  width: 200px;
+  padding: 10px;
+  border-radius: 8px;
+  cursor: pointer;
+
+  @include mq('desktop', $breakpoints) {
+    display: flex;
+    width: 340px;
+  }
+
+  &:hover {
+    background: #fff;
+  }
+
+  &--selected {
+    border-style: solid;
+    border-width: 1px;
```

```suggestion
    border: 1px solid;
```
Swoją drogą możny by ustawić, że `.card` ogólnie ma:
```css
border: 1px solid transparent;
```
a tutaj tylko:
```
border-color: inherit;
```
Wtedy komponent nie będzie "skakał" po zmianie "selected" tak jak teraz. Teraz po zmianie stanu zmienia się rozmiar komponentu o grubość `border`a.